.env

```diff
@@ -64,3 +64,13 @@ CCD_LOGSTASH_SERVICES=all
 #ccd-next-hearing-date-updater
 CCD_NEXT_HEARING_DATE_PASSWORD=Pa55word11
 IDAM_KEY_NEXT_HEARING_UPDATER=AAAAAAAAAAAAAAAA
+
+export XUI_LAUNCH_DARKLY_CLIENT_ID=5de6610b23ce5408280f2268
+
```

Comment:
this is needed for XUI , but may be I can add it on other PR

Reply:
I think so please, if it could be in another PR - and we can add it directly to the `xui-frontend.yaml` instead of in here 😄